src/daemon/server.py

```diff
@@ -234,10 +234,7 @@ async def handle_message(
 
             return None, []
 
-        data = None
-        if "data" in message:
-            data = message["data"]
-
+        data = message["data"]
```

Comment:
This means, any WsRpcMessage has to have a `data` key, or it will fail. I guess the place where the RPC messages get received ensures that, but I haven't check it.

Reply:
Ok, this is handled by a `try`/`except` block in the line 152 of this same file, we can let it crash, or automatically add an empty dict to the data key if there is no data key.

Reply:
Changed to ensure data is always initialized.